### PR TITLE
[BUG] Moodle client return error fix

### PR DESCRIPTION
**Pull request by alex-sheehan-edx**
Base transmitter expects a tuple of (code, body)

**Merge checklist:**
- [x] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [x] `make static` has been run to update webpack bundling if any static content was updated
- [x] `./manage.py makemigrations` has been run
    - Checkout the [Database Migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) Confluence page for helpful tips on creating migrations.
    - *Note*: This **must** be run if you modified any models.
      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
    - This should be run from either a venv with all the lms/edx-enterprise requirements installed or if you checked out edx-enterprise into the src directory used by lms, you can run this command through an lms shell.
        - It would be `./manage.py lms makemigrations` in the shell.
- [x] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [x] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated (see docs/internationalization.rst but also this isn't blocking for merge atm)

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
- [ ] After versioned build finishes in [Travis](https://travis-ci.org/github/edx/edx-enterprise), verify version has been pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)
    - This **must** be done after the version is visible in PyPi as `make upgrade` in edx-platform will look for the latest version in PyPi.
    - Note: the edx-enterprise constraint in edx-platform **must** also be bumped to the latest version in PyPi.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -25,13 +30,11 @@ Unreleased
 
 * Update ``get_service_usernames()`` to read from a list variable (that may not exist).
 
-
```
Comment (alex-sheehan-edx) [inline CHANGELOG.rst:28]: Just want things to be consistent...

File: CHANGELOG.rst [hunk 2]
```diff
@@ -15,6 +15,11 @@ Change Log
 
 Unreleased
 
+[3.8.26] 2020-10-06
```
Comment (binodpant) [inline CHANGELOG.rst:18]: version is wrong

Comment (alex-sheehan-edx) [reply CHANGELOG.rst:18]: 🤦

File: integrated_channels/moodle/client.py [hunk 3]
```diff
@@ -337,7 +336,9 @@ def create_course_completion(self, user_id, payload):  # pylint: disable=unused-
             'grades[0][grade]': completion_data['grade'] * 100
         }
 
-        return self._post(params)
+        # The base learner data transmitter expects a (code, body) tuple to be returned
```
Comment (binodpant) [inline integrated_channels/moodle/client.py:339]: it would be good to see if there is a test that captures this.. any potential to do that?

Comment (alex-sheehan-edx) [reply integrated_channels/moodle/client.py:339]: Uhh, yeah! Can do

File: integrated_channels/moodle/client.py [hunk 4]
```diff
@@ -317,7 +317,6 @@ def delete_content_metadata(self, serialized_data):
             querystring=urlencode(course_ids_to_delete))
         return self._post(params, url)
 
-    @moodle_request_wrapper
```
Comment (taliaedX) [inline integrated_channels/moodle/client.py:320]: I think we need this to ensure a token though. I wonder if there is another way to use the wrapper here to get the right result.

**Review by taliaedX — COMMENTED** [review]
I have concerns about removing the wrapper.

File: CHANGELOG.rst [hunk 5]
```diff
@@ -15,6 +15,11 @@ Change Log
 
 Unreleased
 
+[3.8.29] 2020-10-06
```
Comment (taliaedX) [inline CHANGELOG.rst:18]: _sigh_ so now we're being inconsistent with the dates (not your fault, there's a few examples here)
I know Brandon led the way in getting rid of them, but we seem to be inconsistently adding them back in.
Let's remove it, to try to solidify the trend of removing them :)

File: integrated_channels/moodle/client.py (hunk 4 above)
Comment (alex-sheehan-edx) [reply integrated_channels/moodle/client.py:320]: I think this is doable. One sec.

File: CHANGELOG.rst (hunk 5 above)
Comment (alex-sheehan-edx) [reply CHANGELOG.rst:18]: Mixed feelings, but meh- want to be consistent regardless

**Review by taliaedX — APPROVED** [review]
the new wrapped method and call works for me (I mean as long as it works locally in actual functionality lol)

**Review by binodpant — APPROVED** [review]
lgtm!

**Pull request merged**